src/components/Faq/lang.csv

```diff
@@ -1,307 +1,75 @@
-English,Japanese
-"{
-  ""question"": ""Where can I find the Dev Protocol roadmap?"",
-  ""answer"": ""Best place to find the current roadmap is on our website: https://devprotocol.xyz"",
```

Comment:
Why was this deleted? 👀

Reply:
This is because vinzvinci gave a new [csv format](https://docs.google.com/spreadsheets/d/1VyrRce4qvh2b6rNmbO4QmquQt0eN3_gsAoNhbxWU-jI/edit#gid=0), see #176 for more info.